application/borrower/model.py

```diff
@@ -0,0 +1,18 @@
+from application import db
+
+
+class Borrower(db.Model):
+    __tablename__ = 'borrower'
+
+    id = db.Column(db.Integer, primary_key=True)
+    forename = db.Column(db.String, nullable=False)
+    middlename = db.Column(db.String, nullable=True)
+    surname = db.Column(db.String, nullable=False)
+    dob = db.Column(db.String, nullable=False)
+    gender = db.Column(db.String, nullable=True)
+    phonenumber = db.Column(db.String, nullable=False)
+    address = db.Column(db.String, nullable=False)
+
+    def save(self):  # pragma: no cover
```

Comment:
Pragma comments are used to disable linter checks on a case by case basis. Here @Phemus is trying to avoid a coverage failure by telling coverage to not test this entire function. Why though I'm not sure.